ansible.cfg

```diff
@@ -1,2 +1,6 @@
 [defaults]
 roles_path = ~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:roles
+allow_world_readable_tmpfiles = True
+
+[ssh_connection]
+pipelining = True
```

Comment:
This is a setting that breaks a small % of users running pulp_installer directly from the pulp_installer folder.

But now that the official install instructions are to run from an installed collection, we can do this safely :)

So I like this change, and an FYI & thank you that your Collection work enabled this.